packages/components/src/internal/components/pipeline/utils.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import { AppURL } from '../../url/AppURL';
+import {App, AssayDefinitionModel, AssayUploadResultModel} from '../../../index';
+
+export function getSuccessMsg(
```

Comment:
rename to getAssayImportSuccessMsg? And also maybe this util fits better in assay package? It doesn't seem too much to do with pipeline

Reply:
Renamed! And, moved getAssayImportSuccessMsg and getAssayImportNotificationMsg to assay/utils